supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -363,6 +364,12 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 [[brms]]
 ==== image:images/yes.png[yes] Red Hat JBoss BRMS (noun)
 *Description*: _Red Hat JBoss BRMS_ is a comprehensive platform for business rules management, business resource optimization, and complex event processing (CEP). BRMS stands for Business Rules Management System. Organizations can use Red Hat JBoss BRMS to incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.
+=======
+[discrete]
+[[rhdm]]
+==== image:images/yes.png[yes] Red Hat Decision Manager (noun)
+*Description*: _Red Hat Decision Manager_ is a comprehensive platform for business rules management, business resource optimization, and complex event processing (CEP). Red Hat Decision Manager is a subset of Red Hat Process Automation Manager. Organizations can use Red Hat Decision Manager to incorporate sophisticated decision logic into line-of-business applications and quickly update underlying business rules as market conditions change.
+>>>>>>> 99dc3c6 (remove-obsolete-brms-refs updated doc)
```

Comment:
Marker need to be deleted.

Reply:
I do not see this entry in the [upstream version](https://github.com/redhat-documentation/supplementary-style-guide/blob/main/supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc) of the file. Just want to cehck that it's a new entry?